NStack/strings/ustring.cs

```diff
@@ -129,9 +128,8 @@ public IntPtrUString (IntPtr block, int size, bool copy, Action<IntPtr> releaseF
 					throw new ArgumentException ("Invalid size passed", nameof (size));
 				this.size = size;
 
-				this.copy = copy;
 				if (copy) {
-					this.release = null;
+					this.release = Marshal.FreeHGlobal;
```

Comment:
Right, that is my feeling as well.

I think that what we can do is have a `static Action<IntPtr> None = () => {}` that we assign to, and we can compare at shutdown with.    The reason I say `None` instead of having it point to `Marshal.FreeHGlobal` is that we would avoid the delegate invocation as well.